### ISSUE TITLE
Open sourcing Dispatch

### ISSUE THREAD

**Issue by alulsh** [mapbox/dispatch#97]
Creating an umbrella ticket to discuss and track how to open source Dispatch.

We're dividing tasks into two categories:
1. bare minimum clean up, sanitization, and parameterization that must be done before "flipping the switch" to make Dispatch open source 
2. improvements and fixes that can technically be done after open sourcing but are still critical to Dispatch's success and adoption as an open source project

The first category of tasks won't take long at all, but it'll be important to knock these out of the way quickly so we can spend more of our week working on the features in the second category. We don't necessarily have to open source the repo after all of category 1 is complete - but we want to be able to confidently do this by the end of next week (Friday, November 10th).

@k-mahoney gardened the Dispatch project to move all tasks from category 1 to the "Current Phase". Once those are done (or almost done) we can start moving high priority category 2 items to the "Current phase".

Category 1:
- [x] Code clean up, especially adding a license (https://github.com/mapbox/dispatch/issues/37)
- [x] Make KMS configurable (https://github.com/mapbox/dispatch/issues/95)
- [x] Update GitHub repository parameter (https://github.com/mapbox/dispatch/issues/98)
- [x] Update/add docs for open sourcing (https://github.com/mapbox/dispatch/issues/96) (this is somewhere between category 1 and category 2)
- [x] Dispatch mascot graphic (https://github.com/mapbox/design/issues/209)

Issue already in Category 2 for sure:
- [ ] Low priority PagerDuty alerts (https://github.com/mapbox/dispatch/issues/81)
- [ ] More helpful and descriptive PagerDuty alerts (https://github.com/mapbox/dispatch/issues/89)

**Comment by k-mahoney** [comment]
Dispatch has been open-sourced and Category 2 tasks have been moved to appropriate phases in [project](https://github.com/mapbox/dispatch/projects/1).

**Comment by arunasank** [comment]
This is awesome! 🎉 💃 

cc @mapbox/open-source
